docs/README.md

```diff
@@ -81,20 +83,18 @@ const UISchema = {
 }
 ```
 
-Also note that, regardless if the child is represented in the uiSchema or not, they will still be rendered, to prevent rendering a child use the `hidden` prop in the type.
+Also note that, regardless if the child is represented in the uiSchema or not, they will still be rendered, to prevent rendering a child use the `hidden` prop in the UISchema.
```

Comment:
```suggestion
Also note that, regardless if the child is represented in the `uiSchema` or not, they will still be rendered, to prevent rendering a child use the `hidden` prop in the `UISchema`.
```